### PR TITLE
add zipnerf-pytorch as external method

### PR DESCRIPTION
**Pull request by Jing1Ling**
Add zipnerf-pytorch to the documentation as an external method. 
I’ve submitted a [PR](https://github.com/SuLvXiangXin/zipnerf-pytorch/pull/98) to the [original zipnerf pytorch repo](https://github.com/SuLvXiangXin/zipnerf-pytorch) to support nerfstudio. I'll update link in docs if its merged.

### CODE REVIEW

**Comment by kerrj** [comment]
Cool! maybe we wait until the PR in the other repo is merged before merging this one?

**Comment by Jing1Ling** [comment]
Hi, the PR of another repo has been merged. And I have updated the link in the document:).

File: nerfstudio/configs/external_methods.py
Comment (kerrj) [inline nerfstudio/configs/external_methods.py:202]: It looks like this link doesn't match the github link in the `zipnerf.md` file. Looks like this one points to your fork and the prior one points to the original repo.

Comment (Jing1Ling) [reply nerfstudio/configs/external_methods.py]: Sorry for missing the part in config file when updating Link. Now I've updated it.

File: nerfstudio/configs/external_methods.py
Comment (kerrj) [inline nerfstudio/configs/external_methods.py:207]: looks like there's one more line to update here?

Comment (Jing1Ling) [reply nerfstudio/configs/external_methods.py]: Done! Thanks for reminding.

**Review by kerrj — APPROVED** [review]
Looks good!

**Pull request merged**
